tests/yast2_cmd/yast_lang.pm

```diff
@@ -0,0 +1,27 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: yast language test
+# List languages, set default and secondary languages
+# Maintainer: Michael Grifalconi <mgrifalconi@suse.com>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+    zypper_call "in yast2-country";
+    assert_script_run 'yast language list';
+    assert_script_run 'yast language set lang=en_US languages=en_GB';
```

Comment:
You're just changing the language, but how is it being verified? Also I suggest to use something different from English (spanish or german), as it's being done in https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/locale/keymap_or_locale.pm

But more over, isn't this test duplicating what https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/yast2_gui/yast2_lang.pm is already doing?

Reply:
Let's address the second point first, to clarify this test is meaningful or not:

I agree it is very similar to the gui test, the question would be "Are we interested in testing both command line and gui or is only one enough?"

[This current issue](https://progress.opensuse.org/issues/49286)  is part of [this, where we are asked to test all yast command line options](https://progress.opensuse.org/issues/49244) so I would guess the extra test is a requirement.

To my understanding, the gui test is not scheduled for SLE.

--------------
I agree on setting a different language and I will have a look about validating the change instead of relying on the exit code!

Reply:
Now test is checking output of list languages and making sure the language is changed.

Reply:
Maybe you can reinitialise the session and check that the language really changed in the system.

Reply:
Done and tried another run for 12.4: https://openqa.suse.de/tests/4460579

Reply:
Could someone help me on how can I reinitialize the session?